pkg/tgwriter/tgwriter.go

```diff
@@ -44,6 +44,10 @@ type Error struct {
 }
 
 func (tgw *TgWriter) Write(p []byte) (n int, err error) {
+	if p == nil {
```

Comment:
Does this prevent errors from popping up?

Reply:
It suppresses empty writes from making it onto the wire, which I found can happen when Flush is called. This was leading to the intermittent bug with the `describe` command.